crates/uv/tests/it/lock_conflict.rs

```diff
@@ -2141,7 +2141,7 @@ fn mixed() -> Result<()> {
         wheels = [
             { url = "https://files.pythonhosted.org/packages/32/46/9cb0e58b2deb7f82b84065f37f3bffeb12413f947f9388e4cac22c4621ce/sortedcontainers-2.4.0-py2.py3-none-any.whl", hash = "sha256:a163dcaede0f1c021485e957a39245190e74249897e2ae4b2aa38595db237ee0", size = 29575 },
         ]
-        "###
+        "#
```

Comment:
Do we know where these changes are coming from? Is it an Insta version mismatch on the team? Would be good to understand prior to merging so that they don't oscillate.

Reply:
These are from `cargo insta review`

Reply:
My version is `1.41.1`

Reply:
Yes it's an insta version thing. Old versions used `###` / new versions use `#` when they can, I think.